### DISCUSSION TITLE
YouTube chat dock should be changed.

### DISCUSSION THREAD

**Discussion by Sebasee009** [obsproject/obs-studio#10352] [General]
I would have thought since the introduction of YouTube Live Control Panel in which you also have to log in again, that the chat dock is also logged in, unfortunately I have found that it is not so.
I found out that the YouTube Live Control Panel has a separate browser than the other docks.
I hope it will be changed because I don't want to always have to take the chat as a popup, it has happened to me a few times that I have the wrong one open. You need features and information like poll and moderation activity, so hopefully there will be ways to change the chat dock.

**Comment by sebastian-s-beckmann** [comment]
This sounds like a feature request, which we don't track using GitHub Discussions.
Please put ideas and feature requests on [our ideas page](https://ideas.obsproject.com).

Thank you!
